<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>carControl</class>
 <widget class="QMainWindow" name="carControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>carControl</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(244, 239, 245);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_Door">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: none;
    background: transparent;
    image: url(:/icon/door.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Light">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: none;
    background: transparent;
    image: url(:/icon/light.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_lock">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: none;
    background: transparent;
    image: url(:/icon/lock.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/icon/center.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_airCondition">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: none;
    background: transparent;
    image: url(:/icon/aircondition.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_skylight">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: none;
    background: transparent;
    image: url(:/icon/skylight.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_seatAdjustment">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>150</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: none;
    background: transparent;
    image: url(:/icon/seatAdjustment.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
